File: 1642-water-bottles/1642-water-bottles.cpp
class Solution {
public:
    int numWaterBottles(int numBottles, int numExchange) {
        int eb=numBottles;
        int fb=0;
        int sum=numBottles;
        while(eb>=numExchange){
            sum+=eb/numExchange;
            fb=eb/numExchange;
            eb%=numExchange;
            eb+=fb;
            fb=0;
        }
        return sum;
    }
};

File: 1642-water-bottles/1642-water-bottles.py
class Solution(object):
    def numWaterBottles(self, numBottles, numExchange):
        eb=numBottles
        fb=0
        ans=numBottles
        while(eb>=numExchange):
            ans+=eb//numExchange
            fb=eb//numExchange
            eb%=numExchange
            eb+=fb
            fb=0
        
        return ans
        """
        :type numBottles: int
        :type numExchange: int
        :rtype: int
        """
        